worker/host.go

```diff
@@ -42,11 +42,12 @@ type (
 
 type (
 	host struct {
-		hk         types.PublicKey
-		renterKey  types.PrivateKey
-		accountKey types.PrivateKey
-		fcid       types.FileContractID
-		siamuxAddr string
+		hk           types.PublicKey
+		renterKey    types.PrivateKey
+		accountKey   types.PrivateKey
+		fcid         types.FileContractID
+		siamuxAddr   string
+		fallbackAddr func(ctx context.Context, hk types.PublicKey) string
```

Comment:
I wouldn't make this a function. Instead, it should be set when the `host` is created in `func (w *worker) Host`.